### PR TITLE
Logging in a file

### PR DESCRIPTION
**Pull request by gzamudio**
### Summary
Added log to a file.
Example on HomeController

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ bin
 node_modules
 wwwroot/css
 /appsettings.Development.json
+Logs
```
Comment (gussiciliano) [inline .gitignore]: add final line

File: Startup.cs [hunk 2]
```diff
@@ -71,6 +71,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             {
                 app.UseExceptionHandler("/Home/Error");
             }
+            loggerFactory.AddFile("Logs/NetCoreBootstrapLogs-{Date}.txt", LogLevel.Warning);
```
Comment (gussiciliano) [inline Startup.cs]: i think that the better use Error instead of Warning

File: Controllers/HomeController.cs [hunk 3]
```diff
@@ -9,11 +10,13 @@ public class HomeController : Controller
...
         private readonly DbContextOptions<DataBaseContext> _options;
         private readonly IHtmlLocalizer<HomeController> _localizer;
+        private readonly ILogger<HomeController> _log;
 
-        public HomeController(DbContextOptions<DataBaseContext> options, IHtmlLocalizer<HomeController> localizer)
+        public HomeController(DbContextOptions<DataBaseContext> options, IHtmlLocalizer<HomeController> localizer, ILogger<HomeController> log)
         {
             this._options = options;
             this._localizer = localizer;
+            this._log = log;
```
Comment (gussiciliano) [inline Controllers/HomeController.cs]: is necesary this attribute? maybe with the config in the Startup It's enough

Comment (gzamudio) [reply Controllers/HomeController.cs]: yes, it is necessary in each controller

**Pull request merged**